ciao-scheduler/scheduler.go

```diff
@@ -477,12 +490,35 @@ func (sched *ssntpSchedulerServer) getWorkloadResources(work *payloads.Start) (w
 	return workload, nil
 }
 
+func networkDemandsSatisfied(node *nodeStat, workload *workResources) bool {
+	if node.isNetNode {
+		var matchedNetworksCount int
+		var requestedNetworksCount int
+
+		for _, requestedNetwork := range workload.physNets {
+			requestedNetworksCount++
+			for _, availableNetwork := range node.networks {
+				if requestedNetwork == availableNetwork.NodeIP {
```

Comment:
Yeah it's all slowly headed that direction.  If we would have had the foresight, it would have made sense to start this as a pretty and infinitely extensible interface.  In the meantime though I'm still biasing towards really basic elementary implementation.  I'd like to see a few more use cases come through before we decide we know what the core of the interface should be.